submissions/adekemiAwomolo/dom/newsletter.js

```diff
@@ -0,0 +1,26 @@
+let newsletter = document.getElementById('newsletter');
+newspaper_email = document.getElementById('newsemail');
+emailReg=/^\w+([-]?\w+)*@\w+([-]?\w+)*(\w{2,10})+$/;
+
+function validate_data(field,event){
```

Comment:
No need to pass `field` as a parameter. It is only `newsemail` that is using this function